### PR TITLE
add ARCH arg

### PR DESCRIPTION
**Pull request by keithly**
addresses #21

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,5 +1,6 @@
-ARG AMAZONLINUX_VERSION=2.0.20221210.1-amd64
-FROM public.ecr.aws/amazonlinux/amazonlinux:${AMAZONLINUX_VERSION} as base
+ARG AMAZONLINUX_VERSION=2.0.20221210.1
+ARG ARCH=amd64
```
Comment (alexeyshockov) [inline Dockerfile:2]: IMO you can just use `$TARGETARCH` (which is provided by Docker), to eliminate the custom argument

See https://docs.docker.com/desktop/extensions-sdk/extensions/multi-arch/#adding-multi-arch-binaries

Comment (keithly) [reply Dockerfile:2]: I think that's just for extensions. [Maybe you're thinking of `BUILDPLATFORM` and `TARGETPLATFORM`?](https://docs.docker.com/build/building/multi-platform/#building-multi-platform-images)

**Comment by keithly** [comment]
Relevant docs: 
- https://docs.docker.com/build/building/multi-platform/
- https://www.docker.com/blog/multi-arch-build-and-images-the-simple-way/

There's another factor here: it seems like [Amazon Linux 2 in ECR](https://gallery.ecr.aws/amazonlinux/amazonlinux) doesn't have a manifest for multiplatform like [it does in the docker registry,](https://hub.docker.com/_/amazonlinux/tags?page=1&name=2.0.20221210) and that's why the arch is part of the image name in ECR. I started from the ECR version because that's a more performant and cost choice for AWS users.

I'll have more time to look at this after the holidays, but feel free to make another PR or suggestions.

**Comment by keithly** [comment]
I believe this will work

**Pull request merged**